src/controllers/MigrationController.php

```diff
@@ -296,15 +296,16 @@ public function actionCreate(string $inputTable): int
         foreach ($tables as $tableName) {
             $this->stdout("\n > Generating migration for creating table '{$tableName}' ...", Console::FG_YELLOW);
 
+            $escapedTableName = str_replace('.', '_', $tableName);
```

Comment:
Please change to `$normalizedTableName` and add `use function str_replace;` at the beginning of the file.